src/components/templates/PosterHall/PosterHall.scss

```diff
@@ -29,4 +31,30 @@
     display: flex;
     justify-content: center;
   }
+
+  &__categories {
+    display: flex;
+    justify-content: center;
+    margin: $spacing--md;
+    flex-wrap: wrap;
+  }
+
+  &__category {
+    font-size: $font-size--sm;
+    cursor: pointer;
+    text-transform: capitalize;
+  }
+
+  &__subcategories {
+    display: flex;
+    justify-content: center;
+    margin: $spacing--xs;
+    flex-wrap: wrap;
+  }
+
+  &__subcategory {
+    font-size: $font-size--xxs;
+    cursor: pointer;
+    text-transform: capitalize;
+  }
```

Comment:
It would probably make sense to extract the categories/subcategories stuff into their own component rather than batching it all together in `PosterHall`?